extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.Icons-vector.scss

```diff
@@ -391,6 +391,12 @@
 .oo-ui-icon-media {
 	@include ve-icon(media, wikia);
 }
+.oo-ui-icon-image {
+	@include ve-icon(image, wikia);
+}
+.oo-ui-icon-video {
```

Comment:
There was already a file video.svg, so I named the new file video2.svg

I don't know what the original video.svg icon is used for (it is not referenced in this scss file), it may be referenced elsewhere, so I didn't want to remove it and risk breaking something.